src/toil/test/__init__.py

```diff
@@ -48,6 +54,10 @@ class ToilTest(unittest.TestCase):
     _tempBaseDir = None
     _tempDirs = None
 
+    # Use us-west-2 unless running on EC2, in which case use the region in which
+    # the instance is located
+    AWSRegion = _region() if runningOnEC2() else 'us-west-2'
```

Comment:
The property decorator can only be used with an instance function. Since the ToilClass is not always instantiated when the region is referred to, I made AWSRegion a class variable instead of a property.